myst_nbs/case_studies/regression_discontinuity.myst.md

```diff
@@ -0,0 +1,220 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.7
+kernelspec:
+  display_name: pymc-dev-py39
+  language: python
+  name: pymc-dev-py39
+---
+
+(regression_discontinuity)=
+# Regression discontinuity design analysis
+
+:::{post} April, 2022
+:tags: regression discontinuity, causal inference, quasi experimental design, counterfactuals
```

Comment:
for the tags, maybe `regression` alone should also be added? I think it would be interesting to people browsing the regression tag

Reply:
will do